example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js

```diff
@@ -17,36 +16,31 @@ const {
 const mime = require('mime-types');
 const { loadConfig, createTimestampedTestId, createTestSuffix } = require('../../helpers/testUtils');
 const { waitForModelStatus } = require('../../helpers/apiUtils');
-const config = loadConfig();
-// Make sure that all environment variables are set
-[
-  'AWS_REGION',
-  'EARTHDATA_CLIENT_ID',
-  'EARTHDATA_CLIENT_PASSWORD',
-  'EARTHDATA_PASSWORD',
-  'EARTHDATA_USERNAME',
-  'TOKEN_SECRET'
-].forEach((x) => {
-  if (isNil(process.env[x])) process.env[x] = config[x];
-});
```

Comment:
Why remove this?

Reply:
That's being handled here: https://github.com/nasa/cumulus/blob/CUMULUS-1557-with-buckets-changes/example/spec/helpers/testUtils.js#L89